src/plugins/remote-cache/routes/head-artifact.ts

```diff
@@ -25,15 +25,15 @@ export const headArtifact: RouteOptions<
   schema: artifactsRouteSchema,
   async handler(req, reply) {
     const artifactId = req.params.id
-    const teamId = req.query.teamId ?? req.query.slug // turborepo client passes teamId as slug when --team cli option is used
-    if (!teamId) {
-      throw badRequest(`querystring should have required property 'teamId'`)
+    const team = req.query.team ?? req.query.slug // turborepo client passes team as slug when --team cli option is used
```

Comment:
What about having the old one in place so if someone uses an on older turbo client version, it will still work?
```suggestion
    const team = req.query.teamId ?? req.query.team ?? req.query.slug // turborepo client passes team as slug when --team cli option is used
```

Reply:
Yes 💪!!